src/Service/Symfony/RegistrationRouteListener.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Service\Symfony;
+
+use Buddy\Repman\Service\Config;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+final class RegistrationRouteListener implements EventSubscriberInterface
```

Comment:
it seems to me unnecessary, an additional `if` in the registration controller would suffice, instead of checking it every time for each request